src/articles/2018-01-31-1st-Anniversary/index.tsx

```diff
@@ -138,7 +137,7 @@ const Article = (props: ArticleProps) => {
 
           <Image source={v031} description="Holy Paladin mastery effectiveness calculator v0.3.1 statistics at 18 Mar 2017" /><br />
 
-          The implementation of <ItemLink id={ITEMS.DRAPE_OF_SHAME.id} /> included a large part of the work needed for adding items, so it was possible to quickly add statistics for the similar legendaries Ilterendi, Crown Jewel of Silvermoon and Velen's Future Sight in the next few days.
+          The implementation of <ItemLink id={-1} /> included a large part of the work needed for adding items, so it was possible to quickly add statistics for the similar legendaries Ilterendi, Crown Jewel of Silvermoon and Velen's Future Sight in the next few days.
```

Comment:
```suggestion
          The implementation of Drape of Shame included a large part of the work needed for adding items, so it was possible to quickly add statistics for the similar legendaries Ilterendi, Crown Jewel of Silvermoon and Velen's Future Sight in the next few days.
```

In articles for now we just swap in the name when necessary.